source/common/http/http2/metadata_decoder.cc

```diff
@@ -88,6 +134,34 @@ bool MetadataDecoder::decodeMetadataPayloadUsingNghttp2(bool end_metadata) {
   return true;
 }
 
+bool MetadataDecoder::decodeMetadataPayload(bool end_metadata) {
+  Buffer::RawSliceVector slices = payload_.getRawSlices();
+
+  // Data consumed by the decoder so far.
+  ssize_t payload_size_consumed = 0;
+  for (const auto& slice : slices) {
```

Comment:
very optional/personal preference: I like the explicit type over auto when it's not immediately clear from the line.

Reply:
Done, thanks.